email_service/email_lambda.py

```diff
@@ -53,18 +98,73 @@ def get_user_data(self, conn, data):
         return pd.DataFrame.from_dict(data)
 
 
-class DatabaseConnection:
-    """
-    gets database connection
-    """
+class SendEmail:
 
-    def get_connection(self) -> connection:
-        """Returns a connection to the database movies"""
-        connecting = connect(
-            user=os.environ.get("DB_USER"),
-            password=os.environ.get("DB_PASSWORD"),
-            host=os.environ.get("DB_HOST"),
-            port=os.environ.get("DB_PORT"),
+    def send_email(self, data):
+        print(data)
+        ses = boto3.client('ses',
+                           aws_access_key_id=os.environ.get("ACCESS_KEY"),
+                           aws_secret_access_key=os.environ.get("SECRET_KEY"),
+                           region_name='eu-west-2')
+        ses.send_email(
+            Source='trainee.jonathan.spence@sigmalabs.co.uk',
+            Destination={
+                'ToAddresses': [
+                    'trainee.jonathan.spence@sigmalabs.co.uk'
+                ]
+            },
+            Message={
+                'Subject': {
+                    'Data': 'Hours you should charge your car based on the lowest carbon levels',
+                },
+                'Body': {
+                    'Html': {
+                        'Data': f'Charge your car between {data[0]} and {data[1]}',
+                    }
+                }
+            },
         )
-        connecting.autocommit = True
-        return connecting
+
+
+class ChargeHours:
```

Comment:
Great.

This is a great start, we can make this more comprehensive in subsequent updates.